### PR TITLE
Update 404 page content

### PR DESCRIPTION
**Pull request by avinhurry**
### Context

Updating the 404 page content in line with the prototype

### Screenshot

image

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by avinhurry** [comment]
[Update error pages](https://trello.com/c/T42G3Stf/959-update-error-pages)

**Comment by avinhurry** [comment]
[Update error pages](https://trello.com/c/T42G3Stf/959-update-error-pages)

File: app/views/errors/not_found.html.erb [hunk 1]
```diff
@@ -10,7 +10,7 @@
       If you pasted the web address, check you copied the entire address.
     </p>
     <p class="govuk-body">
-      If the web address is correct or you selected a link or button and you need to speak to someone about this problem, contact the Becoming a Teacher team: <%= bat_contact_mail_to subject: "Page not found" %>
```
Comment (avinhurry) [inline app/views/errors/not_found.html.erb:13]: @Kateharries @simonwhatley A line of text here is different in the [design history](https://bat-design-history.netlify.app/find-teacher-training/updating-error-pages/) and the [prototype](https://publish-courses-prototype.herokuapp.com/404), which should it be?

> If you typed **a** web address, check it is correct.
or 
> If you typed **the** web address, check it is correct.

Comment (avinhurry) [reply app/views/errors/not_found.html.erb:13]: Never mind, I think it's `a` in both of them. I'm confusing myself with all my tabs 😄

**Review by defong — APPROVED** [review]

**Pull request merged**